scripts/aws_star

```diff
@@ -20,6 +22,7 @@ for i in range(args.num_partitions):
                     '--num_partitions {}'.format(args.num_partitions),
                     '--partition_id {}'.format(i),
                     '--input_dirs {}'.format(' '.join(args.input_dirs)),
+                    '--branch {}'.format(args.branch),
```

Comment:
You actually want to move this argument between `evros` and `alignment.run_star_and_htseq` or it will be passed to the alignment script instead of to evros.

Reply:
Right, thanks for catching that!